frontend/src/components/MeetingRoom/ReportIssue/FeedbackForm/FilePicker/FilePicker.tsx

```diff
@@ -63,8 +63,10 @@ const FilePicker = ({
       const screenshotData = await captureScreenshot();
       setImageSrc(screenshotData);
       onFileSelect(screenshotData);
-    } catch (error) {
-      // User cancelled screenshot, no action needed
+    } catch (error: unknown) {
+      if (error instanceof Error) {
+        console.warn(error.message);
+      }
```

Comment:
Note: This will log `Permission denied` in the console

Reply:
To confirm, unless an `Error` is returned, we don't want to log anything?  Not even a "Oops, you broke it, man" message?

Reply:
we could add a little message somewhere that says "oops please give me access permissions next time" but that'll require a UI change which I feel like is outside of the scope of this ticket. I can create a follow up to this one if you feel like that is needed 🙏

Reply:
We don't need a UI change.  Thanks, though.

I was just wondering if we still want to suppress some of the console logging.  I'm guessing is yes 🤷‍♂️ ?